config/remotePlugin.js

```diff
@@ -99,39 +121,40 @@ const routeExtensions = [
     },
   },
 
-  // Import
+  // Applications page.
   {
     type: 'core.page/route',
     properties: {
-      path: '/stonesoup/import',
+      path: '/stonesoup/workspaces/:workspaceName/applications',
       exact: true,
       component: {
-        $codeRef: 'Import',
+        $codeRef: 'Applications',
       },
     },
     flags: {
       required: ['SIGNUP'],
     },
   },
 
-  // Applications page.
+  // Import
   {
     type: 'core.page/route',
     properties: {
-      path: '/stonesoup/applications',
+      path: '/stonesoup/workspaces/:workspaceName/applications/import',
```

Comment:
Shouldn't this be:
```suggestion
      path: '/stonesoup/workspaces/:workspaceName/import',
```

Otherwise I cannot access any page related to my application if I name it `import`

Reply:
updated